src/index.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
   getChangelog: require('./get-changelog'),
   getLatestReleaseCommit: require('./get-latest-release-commit'),
   getNewVersions: require('./get-new-versions'),
+  getWorkspacesPaths: require('./get-workspaces-paths'),
```

Comment:
Let's leave this off for the refactor. We can add it once yarn stops using the globs.